core/server/common/src/main/java/alluxio/underfs/UfsManager.java

```diff
@@ -20,17 +21,46 @@
  * A class that manages the UFS used by different services.
  */
 public interface UfsManager extends Closeable {
+  /** Container for a UFS and the URI for that UFS. */
+  static class Ufs {
```

Comment:
`Ufs` is basically just a form of `Pair<UnderFileSystem, AlluxioURI>`. If use cases arise in the future, we may rename `Ufs` to something like `UnderFileSystemWithMetdata` and use that throughout the codebase. I think we may be able to roll `AlluxioURI` into `UnderFileSystemConfiguration`, but it would touch a lot of code so I think this should be done in a separate PR.